libkbfs/folder_branch_ops.go

```diff
@@ -6073,6 +6073,73 @@ func (fbo *folderBranchOps) onMDFlush(bid BranchID, rev kbfsmd.Revision) {
 	}()
 }
 
+// TeamNameChanged implements the KBFSOps interface for folderBranchOps
+func (fbo *folderBranchOps) TeamNameChanged(
+	ctx context.Context, tid keybase1.TeamID) {
+	fbo.log.CDebugf(ctx, "Team name changed for team %s", tid)
+
+	go func() {
```

Comment:
Given that the goroutine is the whole function, I'd suggest making a method that you can call like so: `go fbo.teamNameChanged(ctx, tid)`. Also, why is it a goroutine?